packages/utils/nft.ts

```diff
@@ -88,4 +90,12 @@ export const nftCardInfoFromStargazeIndexerNft = (
   imageUrl: token.media?.visualAssets?.lg?.url || token.media?.url || undefined,
   name: token.name || token.tokenId || 'Unknown NFT',
   description: token.description || undefined,
+  highestOffer: offerToken
+    ? {
+        offerToken: offerToken,
```

Comment:
```suggestion
        offerToken,
```

style nit: if you want